accounts/views.py

```diff
@@ -689,7 +685,9 @@ def attribution(request):
 
 
 def downloaded_sounds(request, username):
-    user = get_object_or_404(User, username__iexact=username)
+    user = get_user_from_oldusername(username)
+    if user == None:
+        raise Http404
```

Comment:
I like the idea of the util function that gets the user, but this pattern:
```
user = get_user_from_oldusername(username)
    if user == None:
         raise Http404
```
I see everywhere. Why not making `get_user_from_oldusername` raise 404 insetad? In fact, it could be renamed to `get_user_or_404` and I think it would look more elegant everywhere.